pages/installing-page.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, type Locator, type Page } from '@playwright/test';
+
+export class InstallationProgressPage {
```

Comment:
should be call InstallingPage as mentioned in other comment, file name should match with class name.